# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1235078787 18000
#      Thu Feb 19 16:26:27 2009 -0500
# Node ID 8dee145c777d0e8009837e1ecec6560b0ce19cca
# Parent  03b7f618ab3d1197c93416d93e7c1d5723f16df4
fix "Deprecated in version" comments in deprecated fuctions

diff --git a/scripts/deprecated/lchol.m b/scripts/deprecated/lchol.m
--- a/scripts/deprecated/lchol.m
+++ b/scripts/deprecated/lchol.m
@@ -18,17 +18,17 @@
 
 ## -*- texinfo -*-
 ## @deftypefn {Loadable Function} {@var{l} =} lchol (@var{a})
 ## @deftypefnx {Loadable Function} {[@var{l}, @var{p}] =} lchol (@var{a})
 ## This function has been deprecated.  Use @code{chol (@dots{},'lower')}
 ## instead.
 ## @end deftypefn
 
-## Deprecated in version 3.1
+## Deprecated in version 3.2
 
 function varargout = lchol (varargin)
   persistent warned = false;
   if (! warned)
     warned = true;
     warning ("Octave:deprecated-function",
 	     "spfind is obsolete and will be removed from a future version of Octave; please use find instead");
   endif
diff --git a/scripts/deprecated/loadimage.m b/scripts/deprecated/loadimage.m
--- a/scripts/deprecated/loadimage.m
+++ b/scripts/deprecated/loadimage.m
@@ -23,16 +23,18 @@
 ## @var{file}.  The image must be stored in Octave's image format.
 ## @seealso{saveimage, load, save}
 ## @end deftypefn
 
 ## Author: Tony Richardson <arichard@stark.cc.oh.us>
 ## Created: July 1994
 ## Adapted-By: jwe
 
+## Deprecated in version 3.2
+
 function [img_retval, map_retval] = loadimage (varargin)
 
   persistent warned = false;
   if (! warned)
     warned = true;
     warning ("Octave:deprecated-function",
              "loadimage is obsolete and will be removed from a future version of Octave; please use imread instead");
   endif
diff --git a/scripts/deprecated/spatan2.m b/scripts/deprecated/spatan2.m
--- a/scripts/deprecated/spatan2.m
+++ b/scripts/deprecated/spatan2.m
@@ -16,17 +16,17 @@
 ## along with Octave; see the file COPYING.  If not, see
 ## <http://www.gnu.org/licenses/>.
 
 ## -*- texinfo -*-
 ## @deftypefn {Function File} {} spatan2 (@var{y}, @var{x})
 ## This function has been deprecated.  Use @code{atan2} instead.
 ## @end deftypefn
 
-## Deprecated in version 3.1
+## Deprecated in version 3.2
 
 function retval = spatan2 (varargin)
   persistent warned = false;
   if (! warned)
     warned = true;
     warning ("Octave:deprecated-function",
 	     "spatan2 is obsolete and will be removed from a future version of Octave; please use atan2 instead");
   endif
diff --git a/scripts/deprecated/spchol.m b/scripts/deprecated/spchol.m
--- a/scripts/deprecated/spchol.m
+++ b/scripts/deprecated/spchol.m
@@ -18,17 +18,17 @@
 
 ## -*- texinfo -*-
 ## @deftypefn {Loadable Function} {@var{r} =} spchol (@var{a})
 ## @deftypefnx {Loadable Function} {[@var{r}, @var{p}] =} spchol (@var{a})
 ## @deftypefnx {Loadable Function} {[@var{r}, @var{p}, @var{q}] =} spchol (@var{a})
 ## This function has been deprecated.  Use @code{chol} instead.
 ## @end deftypefn
 
-## Deprecated in version 3.1
+## Deprecated in version 3.2
 
 function varargout = spchol (varargin)
   persistent warned = false;
   if (! warned)
     warned = true;
     warning ("Octave:deprecated-function",
 	     "spchol is obsolete and will be removed from a future version of Octave; please use chol instead");
   endif
diff --git a/scripts/deprecated/spchol2inv.m b/scripts/deprecated/spchol2inv.m
--- a/scripts/deprecated/spchol2inv.m
+++ b/scripts/deprecated/spchol2inv.m
@@ -16,17 +16,17 @@
 ## along with Octave; see the file COPYING.  If not, see
 ## <http://www.gnu.org/licenses/>.
 
 ## -*- texinfo -*-
 ## @deftypefn {Function File} {} spchol2inv (@var{u})
 ## This function has been deprecated.  Use @code{chol2inv} instead.
 ## @end deftypefn
 
-## Deprecated in version 3.1
+## Deprecated in version 3.2
 
 function retval = spchol2inv (varargin)
   persistent warned = false;
   if (! warned)
     warned = true;
     warning ("Octave:deprecated-function",
 	     "spchol2inv is obsolete and will be removed from a future version of Octave; please use chol2inv instead");
   endif
diff --git a/scripts/deprecated/spcholinv.m b/scripts/deprecated/spcholinv.m
--- a/scripts/deprecated/spcholinv.m
+++ b/scripts/deprecated/spcholinv.m
@@ -16,17 +16,17 @@
 ## along with Octave; see the file COPYING.  If not, see
 ## <http://www.gnu.org/licenses/>.
 
 ## -*- texinfo -*-
 ## @deftypefn {Function File} {} spcholinv (@var{u})
 ## This function has been deprecated.  Use @code{cholinv} instead.
 ## @end deftypefn
 
-## Deprecated in version 3.1
+## Deprecated in version 3.2
 
 function retval = spcholinv (varargin)
   persistent warned = false;
   if (! warned)
     warned = true;
     warning ("Octave:deprecated-function",
 	     "spcholinv is obsolete and will be removed from a future version of Octave; please use cholinv instead");
   endif
diff --git a/scripts/deprecated/spcumprod.m b/scripts/deprecated/spcumprod.m
--- a/scripts/deprecated/spcumprod.m
+++ b/scripts/deprecated/spcumprod.m
@@ -16,17 +16,17 @@
 ## along with Octave; see the file COPYING.  If not, see
 ## <http://www.gnu.org/licenses/>.
 
 ## -*- texinfo -*-
 ## @deftypefn {Function File} {} spcumprod (@var{x}, @var{dim})
 ## This function has been deprecated.  Use @code{cumprod} instead.
 ## @end deftypefn
 
-## Deprecated in version 3.1
+## Deprecated in version 3.2
 
 function retval = spcumprod (varargin)
   persistent warned = false;
   if (! warned)
     warned = true;
     warning ("Octave:deprecated-function",
 	     "spcumprod is obsolete and will be removed from a future version of Octave; please use cumprod instead");
   endif
diff --git a/scripts/deprecated/spcumsum.m b/scripts/deprecated/spcumsum.m
--- a/scripts/deprecated/spcumsum.m
+++ b/scripts/deprecated/spcumsum.m
@@ -16,17 +16,17 @@
 ## along with Octave; see the file COPYING.  If not, see
 ## <http://www.gnu.org/licenses/>.
 
 ## -*- texinfo -*-
 ## @deftypefn {Function File} {} spcumsum (@var{x}, @var{dim})
 ## This function has been deprecated.  Use @code{cumsum} instead.
 ## @end deftypefn
 
-## Deprecated in version 3.1
+## Deprecated in version 3.2
 
 function retval = spcumsum (varargin)
   persistent warned = false;
   if (! warned)
     warned = true;
     warning ("Octave:deprecated-function",
 	     "spcumsum is obsolete and will be removed from a future version of Octave; please use cumsum instead");
   endif
diff --git a/scripts/deprecated/spdet.m b/scripts/deprecated/spdet.m
--- a/scripts/deprecated/spdet.m
+++ b/scripts/deprecated/spdet.m
@@ -16,17 +16,17 @@
 ## along with Octave; see the file COPYING.  If not, see
 ## <http://www.gnu.org/licenses/>.
 
 ## -*- texinfo -*-
 ## @deftypefn {Loadable Function} {[@var{d}, @var{rcond}] = } spdet (@var{a})
 ## This function has been deprecated.  Use @code{det} instead.
 ## @end deftypefn
 
-## Deprecated in version 3.1
+## Deprecated in version 3.2
 
 function varargout = spdet (varargin)
   persistent warned = false;
   if (! warned)
     warned = true;
     warning ("Octave:deprecated-function",
 	     "spdet is obsolete and will be removed from a future version of Octave; please use det instead");
   endif
diff --git a/scripts/deprecated/spdiag.m b/scripts/deprecated/spdiag.m
--- a/scripts/deprecated/spdiag.m
+++ b/scripts/deprecated/spdiag.m
@@ -16,17 +16,17 @@
 ## along with Octave; see the file COPYING.  If not, see
 ## <http://www.gnu.org/licenses/>.
 
 ## -*- texinfo -*-
 ## @deftypefn {Function File} {} spdiag (@var{v}, @var{k})
 ## This function has been deprecated.  Use @code{diag} instead.
 ## @end deftypefn
 
-## Deprecated in version 3.1
+## Deprecated in version 3.2
 
 function retval = spdiag (varargin)
   persistent warned = false;
   if (! warned)
     warned = true;
     warning ("Octave:deprecated-function",
 	     "spdiag is obsolete and will be removed from a future version of Octave; please use diag instead");
   endif
diff --git a/scripts/deprecated/spfind.m b/scripts/deprecated/spfind.m
--- a/scripts/deprecated/spfind.m
+++ b/scripts/deprecated/spfind.m
@@ -19,17 +19,17 @@
 ## -*- texinfo -*-
 ## @deftypefn {Loadable Function} {} spfind (@var{x})
 ## @deftypefnx {Loadable Function} {} spfind (@var{x}, @var{n})
 ## @deftypefnx {Loadable Function} {} spfind (@var{x}, @var{n}, @var{direction})
 ## @deftypefnx {Loadable Function} {[@var{i}, @var{j}, @var{v}} spfind (@dots{})
 ## This function has been deprecated.  Use @code{find} instead.
 ## @end deftypefn
 
-## Deprecated in version 3.1
+## Deprecated in version 3.2
 
 function varargout = spfind (varargin)
   persistent warned = false;
   if (! warned)
     warned = true;
     warning ("Octave:deprecated-function",
 	     "spfind is obsolete and will be removed from a future version of Octave; please use find instead");
   endif
diff --git a/scripts/deprecated/spinv.m b/scripts/deprecated/spinv.m
--- a/scripts/deprecated/spinv.m
+++ b/scripts/deprecated/spinv.m
@@ -16,17 +16,17 @@
 ## along with Octave; see the file COPYING.  If not, see
 ## <http://www.gnu.org/licenses/>.
 
 ## -*- texinfo -*-
 ## @deftypefn {Function File} {[@var{x}, @var{rcond}] =} spinv (@var{a})
 ## This function has been deprecated.  Use @code{inv} instead.
 ## @end deftypefn
 
-## Deprecated in version 3.1
+## Deprecated in version 3.2
 
 function varargout = spinv (varargin)
   persistent warned = false;
   if (! warned)
     warned = true;
     warning ("Octave:deprecated-function",
 	     "spinv is obsolete and will be removed from a future version of Octave; please use inv instead");
   endif
diff --git a/scripts/deprecated/spkron.m b/scripts/deprecated/spkron.m
--- a/scripts/deprecated/spkron.m
+++ b/scripts/deprecated/spkron.m
@@ -18,17 +18,17 @@
 
 ## -*- texinfo -*-
 ## @deftypefn {Function File} {} spkron (@var{a}, @var{b})
 ## This function has been deprecated.  Use @code{kron} instead.
 ## @end deftypefn
 
 ## Author: jwe
 
-## Deprecated in version 3.1
+## Deprecated in version 3.2
 
 function retval = spkron (varargin)
   persistent warned = false;
   if (! warned)
     warned = true;
     warning ("Octave:deprecated-function",
 	     "spkron is obsolete and will be removed from a future version of Octave; please use kron instead");
   endif
diff --git a/scripts/deprecated/splchol.m b/scripts/deprecated/splchol.m
--- a/scripts/deprecated/splchol.m
+++ b/scripts/deprecated/splchol.m
@@ -19,17 +19,17 @@
 ## -*- texinfo -*-
 ## @deftypefn {Loadable Function} {@var{l} =} splchol (@var{a})
 ## @deftypefnx {Loadable Function} {[@var{l}, @var{p}] =} splchol (@var{a})
 ## @deftypefnx {Loadable Function} {[@var{l}, @var{p}, @var{q}] =} splchol (@var{a})
 ## This function has been deprecated.  Use @code{chol (@dots{},'lower')}
 ## instead.
 ## @end deftypefn
 
-## Deprecated in version 3.1
+## Deprecated in version 3.2
 
 function varargout = splchol (varargin)
   persistent warned = false;
   if (! warned)
     warned = true;
     warning ("Octave:deprecated-function",
 	     "splchol is obsolete and will be removed from a future version of Octave; please use chol instead");
   endif
diff --git a/scripts/deprecated/splu.m b/scripts/deprecated/splu.m
--- a/scripts/deprecated/splu.m
+++ b/scripts/deprecated/splu.m
@@ -20,17 +20,17 @@
 ## @deftypefn {Loadable Function} {[@var{l}, @var{u}] =} splu (@var{a})
 ## @deftypefnx {Loadable Function} {[@var{l}, @var{u}, @var{P}] =} splu (@var{a})
 ## @deftypefnx {Loadable Function} {[@var{l}, @var{u}, @var{P}, @var{Q}] =} splu (@var{a})
 ## @deftypefnx {Loadable Function} {[@var{l}, @var{u}, @var{P}, @var{Q}] =} splu (@dots{}, @var{thres})
 ## @deftypefnx {Loadable Function} {[@var{l}, @var{u}, @var{P}] =} splu (@dots{}, @var{Q})
 ## This function has been deprecated.  Use @code{lu} instead.
 ## @end deftypefn
 
-## Deprecated in version 3.1
+## Deprecated in version 3.2
 
 function varargout = splu (varargin)
   persistent warned = false;
   if (! warned)
     warned = true;
     warning ("Octave:deprecated-function",
 	     "splu is obsolete and will be removed from a future version of Octave; please use lu instead");
   endif
diff --git a/scripts/deprecated/spmax.m b/scripts/deprecated/spmax.m
--- a/scripts/deprecated/spmax.m
+++ b/scripts/deprecated/spmax.m
@@ -17,17 +17,17 @@
 ## <http://www.gnu.org/licenses/>.
 
 ## -*- texinfo -*-
 ## @deftypefn {Mapping Function} {} spmax (@var{x}, @var{y}, @var{dim})
 ## @deftypefnx {Mapping Function} {[@var{w}, @var{iw}] =} spmax (@var{x})
 ## This function has been deprecated.  Use @code{max} instead.
 ## @end deftypefn
 
-## Deprecated in version 3.1
+## Deprecated in version 3.2
 
 function varargout = spmax (varargin)
   persistent warned = false;
   if (! warned)
     warned = true;
     warning ("Octave:deprecated-function",
 	     "spmax is obsolete and will be removed from a future version of Octave; please use max instead");
   endif
diff --git a/scripts/deprecated/spmin.m b/scripts/deprecated/spmin.m
--- a/scripts/deprecated/spmin.m
+++ b/scripts/deprecated/spmin.m
@@ -17,17 +17,17 @@
 ## <http://www.gnu.org/licenses/>.
 
 ## -*- texinfo -*-
 ## @deftypefn {Mapping Function} {} spmin (@var{x}, @var{y}, @var{dim})
 ## @deftypefnx {Mapping Function} {[@var{w}, @var{iw}] =} spmin (@var{x})
 ## This function has been deprecated.  Use @code{min} instead.
 ## @end deftypefn
 
-## Deprecated in version 3.1
+## Deprecated in version 3.2
 
 function varargout = spmin (varargin)
   persistent warned = false;
   if (! warned)
     warned = true;
     warning ("Octave:deprecated-function",
 	     "spmin is obsolete and will be removed from a future version of Octave; please use min instead");
   endif
diff --git a/scripts/deprecated/spprod.m b/scripts/deprecated/spprod.m
--- a/scripts/deprecated/spprod.m
+++ b/scripts/deprecated/spprod.m
@@ -16,17 +16,17 @@
 ## along with Octave; see the file COPYING.  If not, see
 ## <http://www.gnu.org/licenses/>.
 
 ## -*- texinfo -*-
 ## @deftypefn {Function File} {} spprod (@var{x}, @var{dim})
 ## This function has been deprecated.  Use @code{prod} instead.
 ## @end deftypefn
 
-## Deprecated in version 3.1
+## Deprecated in version 3.2
 
 function retval = spprod (varargin)
   persistent warned = false;
   if (! warned)
     warned = true;
     warning ("Octave:deprecated-function",
 	     "spprod is obsolete and will be removed from a future version of Octave; please use prod instead");
   endif
diff --git a/scripts/deprecated/spqr.m b/scripts/deprecated/spqr.m
--- a/scripts/deprecated/spqr.m
+++ b/scripts/deprecated/spqr.m
@@ -19,17 +19,17 @@
 ## -*- texinfo -*-
 ## @deftypefn {Loadable Function} {@var{r} =} spqr (@var{a})
 ## @deftypefnx {Loadable Function} {@var{r} =} spqr (@var{a},0)
 ## @deftypefnx {Loadable Function} {[@var{c}, @var{r}] =} spqr (@var{a},@var{b})
 ## @deftypefnx {Loadable Function} {[@var{c}, @var{r}] =} spqr (@var{a},@var{b},0)
 ## This function has been deprecated.  Use @code{qr} instead.
 ## @end deftypefn
 
-## Deprecated in version 3.1
+## Deprecated in version 3.2
 
 function varargout = spqr (varargin)
   persistent warned = false;
   if (! warned)
     warned = true;
     warning ("Octave:deprecated-function",
 	     "spqr is obsolete and will be removed from a future version of Octave; please use qr instead");
   endif
diff --git a/scripts/deprecated/spsum.m b/scripts/deprecated/spsum.m
--- a/scripts/deprecated/spsum.m
+++ b/scripts/deprecated/spsum.m
@@ -16,17 +16,17 @@
 ## along with Octave; see the file COPYING.  If not, see
 ## <http://www.gnu.org/licenses/>.
 
 ## -*- texinfo -*-
 ## @deftypefn {Function File} {} spsum (@var{x}, @var{dim})
 ## This function has been deprecated.  Use @code{sum} instead.
 ## @end deftypefn
 
-## Deprecated in version 3.1
+## Deprecated in version 3.2
 
 function retval = spsum (varargin)
   persistent warned = false;
   if (! warned)
     warned = true;
     warning ("Octave:deprecated-function",
 	     "spsum is obsolete and will be removed from a future version of Octave; please use sum instead");
   endif
diff --git a/scripts/deprecated/spsumsq.m b/scripts/deprecated/spsumsq.m
--- a/scripts/deprecated/spsumsq.m
+++ b/scripts/deprecated/spsumsq.m
@@ -16,17 +16,17 @@
 ## along with Octave; see the file COPYING.  If not, see
 ## <http://www.gnu.org/licenses/>.
 
 ## -*- texinfo -*-
 ## @deftypefn {Function File} {} spsumsq (@var{x}, @var{dim})
 ## This function has been deprecated.  Use @code{sumsq} instead.
 ## @end deftypefn
 
-## Deprecated in version 3.1
+## Deprecated in version 3.2
 
 function retval = spsumsq (varargin)
   persistent warned = false;
   if (! warned)
     warned = true;
     warning ("Octave:deprecated-function",
 	     "spsumsq is obsolete and will be removed from a future version of Octave; please use sumsq instead");
   endif
diff --git a/scripts/deprecated/str2mat.m b/scripts/deprecated/str2mat.m
--- a/scripts/deprecated/str2mat.m
+++ b/scripts/deprecated/str2mat.m
@@ -26,16 +26,18 @@
 ## This function is modelled after @sc{Matlab}.  In Octave, you can create
 ## a matrix of strings by @code{[@var{s_1}; @dots{}; @var{s_n}]} even if
 ## the strings are not all the same length.
 ## @end deftypefn
 
 ## Author: Kurt Hornik <Kurt.Hornik@wu-wien.ac.at>
 ## Adapted-By: jwe
 
+## Deprecated in version 3.2
+
 function retval = str2mat (varargin)
   persistent warned = false;
   if (! warned)
     warned = true;
     warning ("Octave:deprecated-function",
              "str2mat is obsolete and will be removed from a future version of Octave; please use char instead.");
   endif
 
